### ISSUE TITLE
Skip fixing for specific rules

### ISSUE THREAD

**Issue by davidgm0** [DavidAnson/markdownlint#995]
Hi, I am using `markdownlint-cli`, but I believe that this belongs to this repo.

I was giving a try to fixing (with `--fix), and the fixing is in my opinion overly optimistic. If I make a typo like

```md
[Github] 

..

<!-- links -->

[Githuv]
```

`--fix` will remove [Githuv] as a link and succeed. I think it can be quite common to make a typo in a link, specially with upper-lower case. I'm using this with git hooks, so this for me means silently removing links.

I haven't found an option to skip `--fix` for specific rule. Would it be possible to implement this? Or to remove the option to fix this rule, but I imagine that some people find it useful.

**Comment by DavidAnson** [comment]
I need some more information about your scenario and configuration, please. As you can see in the demo, missing shortcut links are not reported as an issue in the default configuration due to ambiguity: https://dlaa.me/markdownlint/#%25m%23%20Issue%20995%0A%0A%5BGitHub%5D%0A%0A%5BBroken%5D%0A%0A%5BGitHub%5D%3A%20https%3A%2F%2Fgithub.com%0A

**Comment by davidgm0** [comment]
I see, I'll take a look and see what's up
